docs/README-Developer.md

```diff
@@ -207,3 +208,14 @@ Maintainers: Evann Wu (evannw@andrew.cmu.edu), Lauren Li (lauren.li@ibm.com), Kr
    3. Affiliation with Zowe
 
 3. You should receive a response within 3 business days and be added to the Zanata Zowe VS Code Extension project. Click [here](http://docs.zanata.org/en/release/user-guide/translator-guide/) for more information about how to use Zanata to translate.
+
+## Packaging
+
+Run this command to create packages in the "dist" directory:
+`yarn workspace vscode-extension-for-zowe package`
```

Comment:
should this line be `yarn package` to get all 3 packaged and in the dist dir.  Having `workspace vscode-extension-for-zowe` will only package the single vsix